src/components/sdk/VideoTileGrid/VideoTileGrid.stories.mdx

```diff
@@ -5,7 +5,7 @@ import { VideoTileGrid } from './';
 
 # VideoTileGrid
 
-The `VideoTileGrid` component renders all meeting session video tiles in a responsive grid layout. This includes the local tile, remote tiles, and content share tile.
+The `VideoTileGrid` component renders all meeting session video tiles in a responsive grid layout. This includes the local tile, remote tiles, and content share tile. By default a user joins without video, so in order to see the VideoTileGrid there must be at least one video tile being shared. 
```

Comment:
Can we add something about starting video?

`To start sharing a video, see the "VideoButton or w/e" component`

Reply:
Got it, will update.